CMakeLists.txt

```diff
@@ -44,7 +44,14 @@ if(CMAKE_COMPILER_IS_GNUCXX OR ${CMAKE_CXX_COMPILER_ID} MATCHES Clang)
   add_definitions(-Wall)
 endif()
 
-option(DISABLE_SSE "Disable SSE optimizations" OFF)
+check_c_source_compiles(
+    "#if !defined(__x86_64) && !defined(__i386__) \\
+    && !defined(_M_IX86) && !defined(_M_AMD64)
+    #error not x86
+    #endif
+    int main(){return 0;}"
+    HAVE_X86)
+set(DISABLE_SSE "Disable SSE optimizations" ${HAVE_X86})
```

Comment:
This is broken in multiple ways.

1) The `HAVE_X86` test fails under MSVC because the `\\` line continuation is invalid (GCC allows it).
2) The original `option` description was left in the `set`, which creates a list variable containing `Disable SSE optimizations` if the test fails and `Disable SSE optimizations;1` if it passes.  This means the `if(NOT DISABLE_SSE)` test in `libde265/CMakeLists.txt` is always evaluated as false, and the SSE code is never compiled.
3) Even ignoring the above, the logic is backwards.  It should disable SSE if the test fails, not if it succeeds.